docs(app-branding-demo): adding an app title generator for navbar

sheaivey opened the pull request:
I created a tool to generate the application title text people frequently ask for.

I was not sure the exact font used to create these titles. But I found one that is pretty close. `Carnas Light`
It should be pretty easy to plug in the actual font-family found in `app-branding-demo.scss` and then updating all the references to use it.

The new title generator can be found in Foundations -> Brand -> App Title Generator

![Screen Shot 2024-02-15 at 5 54 06 PM](https://github.com/HealthCatalyst/Fabric.Cashmere/assets/3945263/538bb21e-d796-4459-a575-ed61fed21d99)

sheaivey commented:
Updating to use `Flexo-Regular` font. It appears to be the actual font used in the past.

corykon commented:
@sheaivey THANK YOU! This is awesome. Looks like an great solution.

Couple points of feedback
- The primary normal text should be a slightly gray color - #f4f3f3. The secondary bold text is just plain old #fff
- Would it be possible to base64 encode as png instead of a gif? The analytics teams always ask for pngs, but a gif may be just fine. I'll ask ASO about that
- An awesome bonus would be if we could spit out SVGs for webapps, but that sounds like a much taller order.

aaronbrethorst commented:
This repo purports to convert text to svg paths: https://github.com/shrhdk/text-to-svg

sheaivey commented:
- I updated the normal text color to be `#f4f3f3` and the bold text to `#ffffff`. 
- I also added output format selector. Currently canvas only supports png or jpeg.

It would be fun in the future to give the `text-to-svg` library a try that @aaronbrethorst mentioned.

health-catalyst commented:
:tada: This PR is included in version 16.1.3 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@healthcatalyst/cashmere/v/16.1.3)
- [GitHub release](https://github.com/HealthCatalyst/Fabric.Cashmere/releases/tag/v16.1.3)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
